app/controllers/assignments_controller.rb

```diff
@@ -3,14 +3,17 @@ class AssignmentsController < ApplicationController
   before_action :authorize_user
   after_action :verify_authorized
 
+  def show
```

Comment:
I was expecting `internal_contacts_controller` what do you think - are there advantages to having this as the assignment show action?

Reply:
No advantage, I was just piggybacking on an existing one for convenience!